### DISCUSSION TITLE
Why do the docs ask you to nest an anchor tag inside a <Link /> component?

### DISCUSSION THREAD

**Discussion by burrack** [vercel/next.js#17139] [Help] [answered]
When I inspected the page I saw that the Link component is adding an anchor tag automatically? 
[docs](https://nextjs.org/learn/basics/navigate-between-pages/link-component)

**Comment by jamesmosier** [comment, marked as answer]
The `next/link` component clones the child and applies the specified props to it. 

https://github.com/vercel/next.js/blob/4ba768e22c56141d5488f8ee1f1d78d703b0fbfd/packages/next/client/link.tsx#L337

There is a deprecated piece of code that if the child of `<Link>` is a string they would wrap it in `<a>` tags, but as mentioned this is no longer preferred and may not always be supported.

https://github.com/vercel/next.js/blob/4ba768e22c56141d5488f8ee1f1d78d703b0fbfd/packages/next/client/link.tsx#L289-L291

**Comment by sergioviniciuss** [comment]
@jamesmosier  but what about this piece of code here:
https://github.com/vercel/next.js/blob/canary/packages/next/client/link.tsx#L292

Check the comment on line 292. It's also doing it.

I'm still confused if we can drop the <a> tag inside a link component, or if we should keep it.

**Reply by jamesmosier** [reply]
That comment to me still reads that the child needs to be an `<a>` tag. It does not automatically create the `<a>` tag for you in that code, rather it is just passing the `href` to your child `<a>`.

It is still recommended/required that you put your own `<a>` tag within the `<Link>` tag per the docs https://nextjs.org/docs/api-reference/next/link
